dbaas/dbaas.go

```diff
@@ -82,6 +83,21 @@ func runCmd(cmd string, args ...string) ([]byte, error) {
 	return o, nil
 }
 
+func runEnvCmd(Env []string, cmd string, args ...string) ([]byte, error) {
+	cli := exec.Command(cmd, args...)
+	cli.Env = os.Environ()
+
+	for _, envVar := range Env {
+		cli.Env = append(cli.Env, envVar)
+	}
```

Comment:
instead of 
```go
	cli.Env = os.Environ()

 	for _, envVar := range Env {
		cli.Env = append(cli.Env, envVar)
	}
```
you may use just
```go
cli.Env = append(os.Environ(), Env...)
```

Reply:
done